packages/manager/src/features/Linodes/LinodesDetail/LinodeNetworking/LinodeNetworkingActionMenu.tsx

```diff
@@ -59,6 +59,14 @@ export const LinodeNetworkingActionMenu = (props: Props) => {
     ? 'Linodes must have at least one public IP'
     : undefined;
 
+  const getAriaLabel = (): string => {
+    if (ipAddress && 'address' in ipAddress) {
```

Comment:
now that ipAdddress won't be undefined,, we may be okay to do this:
```suggestion
    if ('address' in ipAddress) {
```